src/calculateRentalCost.js

```diff
@@ -27,8 +27,21 @@
  * @return {number}
  */
 
+calculateRentalCost(7);
+
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const rentalCost = days * dayPrice;
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
```

Comment:
keep constants around constants
```suggestion
  const dayPrice = 40;
  const threeDayDiscount = 20;
  const sevenDayDiscount = 50;
  
  const rentalCost = days * dayPrice;
```